rrfs-test/ush/plot_jdiag.all.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+import os
+import sys
+import cartopy.crs as ccrs
+from cartopy.mpl.ticker import (LongitudeFormatter, LatitudeFormatter,
+                                LatitudeLocator, LongitudeLocator)
+from collections.abc import Iterable
+import numpy as np
+import matplotlib
+matplotlib.use('AGG')
+import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import matplotlib.axes as maxes
+import h5py as h5
```

Comment:
This wasn't in the EVA/hera environment. Can we use `import netCDF4 as nc`?

Reply:
I had h5py in my environment when I was making the plots but didn't realize it was not in EVA. Will modify to use netcdf instead. Thanks!